test/mocha/mock.data.js

```diff
@@ -0,0 +1,76 @@
+const {config: {constants}} = require('bedrock');
+
+const mock = {};
+module.exports = mock;
+
+const didContexts = [
+  constants.DID_CONTEXT_URL,
+  constants.VERES_ONE_CONTEXT_V1_URL
+];
+
+const credentials = mock.credentials = {};
+credentials.alpha = {
+  "@context": [
+    constants.CREDENTIALS_CONTEXT_V1_URL, {
+      "ex1": "https://example.com/examples/v1",
+      "AlumniCredential": "ex1:AlumniCredential",
+      "alumniOf": "ex1:alumniOf"
+    }
+  ],
+  "id": "http://example.edu/credentials/58473",
+  "type": ["VerifiableCredential", "AlumniCredential"],
+  "issuer": "did:test:issuer:foo",
+  "issuanceDate": new Date().toISOString(),
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "alumniOf": "Example University"
+  }
+};
+
+const presentations = mock.presentations = {};
+
+presentations.alpha = {
+  "@context": [constants.CREDENTIALS_CONTEXT_V1_URL],
+  "type": ["VerifiablePresentation"],
+  "verifiableCredential": [],
+};
+
+const privateDidDocuments = mock.privateDidDocuments = {};
+
+privateDidDocuments.alpha = {
+  "@context": didContexts,
+  "id": "did:v1:test:nym:z279yHL6HsxRzCPU78DAWgZVieb8xPK1mJKJBbP8T2CezuFY",
+  // FIXME should generate a new key for this.
+  "assertionMethod": [
```

Comment:
If this didn't end up getting used in the stuff we did @aljones15, let's remove it.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/1/commits/649b8d87e76e10658c6c700878807b883d8587e1